compiler/one-cmds/tests/onnx_legalize_run_compare.py

```diff
@@ -0,0 +1,76 @@
+import onnxruntime as rt
+import onnx
+import sys
+import os
+import numpy as np
+
+
+def _generate_inputs(model):
+    inputs = {}
+    for input in model.graph.input:
+        # check if elem type is float32
+        assert (input.type.tensor_type.elem_type == 1)
+        input_shape = []
+        for dim in input.type.tensor_type.shape.dim:
+            input_shape += [dim.dim_value]
+        inputs[input.name] = np.random.random(input_shape).astype(np.float32)
+    return inputs
```

Comment:
(optional) It would be helpful to let readers know what input's spec is like with comment.